thoth/storages/graph/postgres.py

```diff
@@ -4163,6 +4163,9 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
         os_name = document["result"]["operating-system"]["name"]
         os_version = document["result"]["operating-system"]["version_id"]
+        cuda_version_txt = document["result"]["cuda-version"]["/usr/local/cuda/version.txt"]
+        cuda_version_nvcc = document["result"]["cuda-version"]["nvcc_version"]
+        cuda_version = cuda_version_txt if cuda_version_txt == cuda_version_nvcc else None
```

Comment:
Let's raise an exception here so we get notified about this fact and the sync will be aborted - at least now in the initial implementation.

Reply:
Thanks!